gitlab/const.py

```diff
@@ -51,3 +54,5 @@
 
 # specific project scope
 SEARCH_SCOPE_PROJECT_NOTES = "notes"
+
+USER_AGENT = "{}/{}".format(__title__, __version__)
```

Comment:
I guess we could already use f-strings here. Not a blocker though.

Reply:
@nejch Should I merge or do you want to change this still?

Reply:
@max-wittig I was thinking about this as well but in the end decided against since, if I remember correctly I didn't see any f-strings in the production code (only tests), so I thought maybe some poor soul is still on 3.5 (even if unsupported officially) and a minor/patch version bump would be confusing if it started failing all of a sudden 😁 

Maybe a follow-up issue to also clear all old-style % operators as well and just turn everything to f-strings consistently?

Reply:
Sure.